DuckDuckGo/AutofillLoginSettingsListViewController.swift

```diff
@@ -236,7 +236,8 @@ final class AutofillLoginSettingsListViewController: UIViewController {
     }
 
     private func presentDeleteConfirmation(for title: String) {
-        ActionMessageView.present(message: UserText.autofillLoginListLoginDeletedToastMessage(for: title),
+        ActionMessageView.present(message: title.isEmpty ? UserText.autofillLoginListLoginDeletedToastMessageNoTitle
+                                                         : UserText.autofillLoginListLoginDeletedToastMessage(for: title),
```

Comment:
Small nit, maybe setting the title as a const would be easier to read, but mainly because it wouldn't break Xcode's automatic indentation if it's ran in this block.

Something like this:
```swift
let title = title.isEmpty ? UserText.autofillLoginListLoginDeletedToastMessageNoTitle : UserText.autofillLoginListLoginDeletedToastMessage(for: title)

ActionMessageView.present(message: title,
                          actionTitle: UserText.actionGenericUndo,
                          presentationLocation: .withoutBottomBar,
                          onAction: {
    self.viewModel.undoLastDelete()
}, onDidDismiss: {
    self.viewModel.clearUndoCache()
})
```